packages/plugin/src/generators/app/templates/scripts/generate-manifest.js

```diff
@@ -0,0 +1,47 @@
+(async () => {
+    const fs = require('fs');
+
+    //1. Read from the "widgets" folder
+    const folderName = 'src/widgets';
+    const getWidgetProperties = async (widgetsList) => {
+        const temp = [];
+        console.log('');
+        for await (const widget of widgetsList) {
+            try {
+                const widget_folder_path = folderName + '/' + widget;
+                // eslint-disable-next-line node/no-unsupported-features/es-syntax
+                let WidgetProperties = await import('../' + widget_folder_path + '/WidgetProperties.json', {
+                    assert: { type: 'json' },
```

Comment:
@bradwray  is this what you meant to remove in your suggestion to fix the warning? I don't remember for sure if removing this assertion will still work (i.e. will the json still be imported or not). But yea, try and see if it does.

Reply:
Yeah that comma after the file path shows a parsing error. And the terminal complains that running assertions on an import is an experimental feature.

Reply:
@bradwray i meant we should check if we remove the assert, will the import still work as expected for jsons? If it does, im fine with removing the assertion.